Makefile

```diff
@@ -819,6 +819,7 @@ PREVIOUS_TAG ?= $(shell git tag -l | grep -E "^v[0-9]+\.[0-9]+\.[0-9]+$$" | sort
 RELEASE_ALIAS_TAG := $(PULL_BASE_REF)
 RELEASE_DIR := out
 RELEASE_NOTES_DIR := _releasenotes
+# can be set directly if needed, e.g., export USER_FORK=<personal GitHub hanlde>
```

Comment:
What about moving this into the release notes section instead?

Reply:
I thought it would be confusing to explain it in the release task doc without explaining where and how it is used in the Makefile, but open to ideas/suggestions

Reply:
https://github.com/kubernetes-sigs/cluster-api/pull/8597#discussion_r1184017380